google/ggrc-core
# GGRC-4703 Cycle task PUTs should take LESS than 1minute on RMC

# Issue description

We need to investigate why cycle task updates take a very long time on ggrc-rmc.

# Steps to test the changes

1. Create CycleTasks with couple of thousands of Controls related to it
2. Update this CycleTask
3. Check benchmark values for 'Update memcache before commit for collection PUT' benchmark
(Without the change 8.9s after the change 0.5s)

# Solution description

Eliminated unnecessary data collecting by using one query for fetching objects related to CycleTaskGroupObjectTask

# Sanity checklist




# PR Review checklist
